modules/nw-metallb-frr-k8s-configuration-crd.adoc

```diff
@@ -0,0 +1,406 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-metallb-frrconfiguration-crd_{context}"]
+= Configuring the FRRConfiguration CRD
+
+The following section provides reference examples that use the `FRRConfiguration` custom resource (CR).
+
+[id="nw-metallb-frrconfiguration-crd-routers_{context}"]
+== The routers field
+
+You can use the `routers` field to configure multiple routers, one for each Virtual Routing and Forwarding (VRF) resource.
+For each router, you must define the Autonomous System Number (ASN). 
+You can also define a list of Border Gateway Protocol (BGP) neighbors to connect to, as in the following example:
+
+.Example FRRConfiguration CR
```

Comment:
The example is a little confusing. Are you showing a VRF example or just how to define a list of BGP neighbors?  If its a vrf use different ASN for each router. And why are you using the different ports?

Reply:
The example is about the default VRF. So we might separate the first line to the second and state that this is for the default VRF, otherwise we'd need multiple VRFs and router stanzas (which we don't have here).

The ports are legit, it shows how you can define a different peer port.

Reply:
Thanks for the suggestion @fedepaol 
@gkopels I've separated the text as suggested. I think that clarifies that the example isn't about VRF, just how to define BGP neighbours.

Reply:
Looks good